test/migrations/4-test4.coffee

```diff
@@ -0,0 +1,4 @@
+module.exports.id = "4-test4"
```

Comment:
Why is this file needed? Does it demonstrate any new concept or the edge case?
Also the ID style is different, others don't have the order number in the ID.